visualization/app/codeCharta/ui/codeMap/threeViewer/floorLabels/floorLabelDrawer.ts

```diff
@@ -111,32 +111,37 @@ export class FloorLabelDrawer {
 		planeMesh.rotateX((90 * Math.PI) / 180)
 
 		// Position plane over the map
-		const liftToPreventZFighting = 10
-		plane.translate(scaledMapWidth / 2, scaledMapHeight / 2, -2.01 * (floorLevel + 1) - liftToPreventZFighting)
+		const liftToPreventZFighting = 2
```

Comment:
The lift is not ideal, but what is the reason why you changed this?
The labels are disappearing more quickly now.

Reply:
Only when changing the height though, which is proven to be a bug as mentioned in [#2763 2764](https://github.com/MaibornWolff/codecharta/issues/2764). Changing the lift to 2 is just a visual change, that is apparent when having maps with high scales and floor labels on the correct position. The floating with the value set to 10 is very apparent